src/psij/web/js/lib.js

```diff
@@ -385,7 +488,10 @@ var globalMethods = {
     },
     getDayData: getDayData,
     breadcrumbs: function() {
-        var b = [{text: CUSTOMIZATION.projectName + " Tests", href: "summary.html"}];
+
+        var projName = PS.isSDKPage() ? "SDK" : "PSI/J";
```

Comment:
I thought the psij and sdk page both use the same codebase.  are you suggesting that we should be loading sdk and psij from two different code deployments?

Reply:
customization.js, as the name suggests, is not part of the common code base. It is where we currently differentiate between PSI/J and SDK.

I think we discussed this a bit in #24 and, at that time, you convinced me that it would be a good idea to store project specific strings in a single place (like customization.js). At that time I was saying that, for the purpose of the introductory text, we could have two different html pages or some kind of templating for the text such that text changes are close to the view that represents them. I'm still somewhat in-between in that the text is part of the view, but I'm also leaning towards what you were saying about having all differences in one place.

However, code that explicitly mentions the SDK in a function name is a bit of a different thing and I do not think it is either what you were proposing then or what I was proposing then.

Specifically, `CUSTOMIZATION.projectName` is a more reasonable solution than `PS.isSDKPage() ? "SDK" : "PSI/J"`.

Reply:
okay but then we will need to have one customization.js for psij and one for SDK.   so do you just want me to make like: psij_customization.js and sdk_customization.js or did you have something else in mind?

Reply:
Let's do that. Have `instance/customization-psij.js` and `instance/customization-sdk.js`. Then, on the deployments, we do a symbolic link from either to `instance/customization.js` and we only load `instance/customization.js` in html files.

Reply:
I'm a bit worried that this won't work the same in dev as in prod hence opening us up to possibility of bugs, plus the additional step of setting up the symbolic link.

How about we do this:
<script>
var custSuffix = window.location.href.subdomain;
var custToLoad = "customization-" + custSuffix + ".js";
//  Now load the appropriate file:
</script>

Reply:
> I'm a bit worried that this won't work the same in dev as in prod

Why would that be the case?

> the additional step of setting up the symbolic link

It's a one time step.

> <script> var custSuffix = window.location.href.subdomain; var custToLoad = "customization-" + custSuffix + ".js"; // Now load the appropriate file: </script>

We don't know for sure that we'll keep the current url scheme.

Reply:
by what mechanism will the Link be different from psij to sdk?

Reply:
> by what mechanism will the Link be different from psij to sdk?

by running different `ln -s` commands when first setting up a deployment.

Reply:
so, it sounds like we'll need to create two locations, one for psij and one for sdk and two ln -s     and each subdomain will need to point to each codebase.  and symbolic links can break.  just seems more complicated and error prone.

Reply:
> it sounds like we'll need to create two locations, one for psij and one for sdk [...] and each subdomain will need to point to each codebase.

That's already the case. We have two containers each with an instance of the web service and database.

> symbolic links can break

I have not heard of this before. How do symbolic links break?